lang/en/docs/tutorials/dft/electronic/gw-vasp-bg.json

```diff
@@ -0,0 +1,99 @@
+{
+    "descriptionLinks": [
+        "Calculate Electronic Band Gap via GW Approximation: https://docs.exabyte.io/tutorials/dft/electronic/gw-vasp-bg/"
+    ],
+    "description": "We illustrate in this animation how the results for a band gap computation on crystalline silicon can be further improved and made to be more accurate through the use of the GW Approximation.",
+    "tags": [
+        {
+            "...": "../../../metadata/general.json#/tags"
+        },
+        {
+            "...": "../../../models-directory/dft.json#/tags"
+        },
+        {
+            "...": "../../../software-directory/modeling/vasp.json#/tags"
+        },
+        {
+            "...": "../../../properties-directory/non-scalar/band-gaps.json#/tags"
+        }
+    ],
+    "title": "Exabyte.io Tutorial: Band Gap via GW Approximation",
+    "youTubeCaptions": [
+        {
+            "text": "Band Gap via GW Approximation tutorial.",
+            "endTime": "00:00:03.000",
+            "startTime": "00:00:00.000"
+        },
+        {
+            "text": "Create a simulation, by navigating to the job designer page.",
+            "endTime": "00:00:07.000",
+            "startTime": "00:00:03.500"
+        },
+        {
+            "text": "Then, <break time='0.5' /> open the workflow tab, and select from the list the 'D1-GW0-BG' workflow, whose naming convention is explained in the tutorial text linked under the video description.",
+            "endTime": "0:00:19.000",
+            "startTime": "0:00:07.500"
+        },
+        {
+            "text": "We now explore the subworkflow structure making up the overall workflow.",
+            "endTime": "0:00:23.000",
+            "startTime": "0:00:19.500"
+        },
+        {
+            "text": "Try, for example, opening the 'SCF many bands' subworkflow, used to perform a calculation over a large number of unoccupied bands.",
+            "endTime": "0:00:33.000",
+            "startTime": "0:00:24.000"
+        },
+        {
+            "text": "Then, open the only unit component, and inspect the <sub alias='WASP'>VASP</sub> input files displayed at the bottom. <break time='1.0' /> The <sub alias='N BANDS'>NBANDS</sub> tag sets the number of bands to a large value. <break time='7.0' />",
+            "endTime": "0:00:44.000",
+            "startTime": "0:00:33.500"
+        },
+        {
+            "text": "Let us also inspect the contents of the main GW step of the computation, by opening the G0W0 step subworkflow. <break time='1.0' /> Again, let us review the <sub alias='WASP'>VASP</sub> input files.",
+            "endTime": "0:01:02.000",
+            "startTime": "0:00:50.000"
+        },
+        {
+            "text": "The ALGO tag allows us to choose the 'G0W0' version of the GW approximation, which is the simplest and computationally the most efficient one.",
```

Comment:
Stop using so many superlatives everywhere:

"The ALGO tag allows us to choose the 'G0W0' version of the GW approximation, the simplest one."